index.html

```diff
@@ -12,29 +12,78 @@
             </div>
 
             <!-- Page NAV (Menu) block -->
-            <nav class="flex-box nav-bar">
+            <nav class="flex-box horizontal nav-bar black-fg white-bg">
                 <ul>
-                  <li>About me</li>
+                  <li>
+                      <a href="index.html" target="_self" rel="">
+                          <i class="fa-solid fa-bars fa-1x blue-fg white-bg"></i>
+                      </a>
+                  </li>
                 </ul>
             </nav>
 
         </header>
-
+ 
 
         <!-- Page main body block -->
-        <div class="flex-box main-body">
+        <div class="flex-box vertical main-body">
+
+            <!-- First section -->
+            <section class="flex-box vertical flex-center transparent-bg black-fg" id="section_1" >
+
+                <h1 class="main-text-color main-title-style" >I am Marianne<br/>Glad to see<br/>you!</h1>
+
+                <p class="main-text-color" >I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+
+                <div class="flex-box vertical flex-center">
+                    <h3 class="lc-button-text-color">LET'S CONNECT</h3>
+                    <ul class="flex-box horizontal flex-start">
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_1" src="@images/social_logo_1.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_2" src="@images/social_logo_2.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_3" src="@images/social_logo_3.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_4" src="@images/social_logo_4.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_5" src="@images/social_logo_5.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </section>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you review the images again and try to use appropriate font families for your `p` and your `h3` - as well as use the correct colors on your `h3` (Let's connect) as seen below:

![image](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/99715640/2b97dfb9-32e9-424c-be21-bfe82526d508)